.travis.yml

```diff
@@ -2,20 +2,13 @@ language: python
 cache: pip
 
 python:
-  - "3.5"
-  - "2.7"
+  - "3.7"
 
 env:
   - DJANGO="django>=1.11,<1.12"
   - DJANGO="django>=2.0,<2.1"
   - DJANGO="django>=2.1,<2.2"
-
-matrix:
-  exclude:
-    - python: "2.7"
-      env: DJANGO="django>=2.0,<2.1"
-    - python: "2.7"
-      env: DJANGO="django>=2.1,<2.2"
+  - DJANGO="django>=3.0"
```

Comment:
this should be `- DJANGO="django>=3.0,<3.1"

Reply:
@nemesisdesign Ok I looked at django-allauth seems like their latest version doesnt support djanngo 3 and it is installed here `pip install https://github.com/openwisp/openwisp-users/tarball/master`

Reply:
Try with the development version @waleko